common.sh

```diff
@@ -34,9 +34,10 @@ fi
 source $CONFIG
 
 #
-# See https://origin-release.svc.ci.openshift.org/ for release details
+# See https://openshift-release.svc.ci.openshift.org for release details
 #
-export OPENSHIFT_RELEASE_IMAGE="${OPENSHIFT_RELEASE_IMAGE:-registry.svc.ci.openshift.org/ocp/release:4.2}"
+LATEST_CI_IMAGE=$(curl https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4.3.0-0.ci/latest | jq ".pullSpec")
```

Comment:
This looks good to me, although I'm concerned how many times we'll hit the API.  Because of how make runs each script individually, exported variables aren't carried over. We recalculate everything in common.sh multiple times during a run.

I've been thinking about this for a while, actually, and I was thinking what if we source common.sh before we run `make`? That is, have a target that sources common.sh and then calls make itself. Is that too crazy? See https://github.com/stbenjam/dev-scripts/commit/af9556503a72fbdacf70dda6492c598bd7e5b823

If we had that, then changing this line to:
```
export LATEST_CI_IMAGE=${LATEST_CI_IMAGE:-$(curl https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4.3.0-0.ci/latest | jq ".pullSpec")}
```

We would avoid many curl calls and just have one.

Reply:
Perhaps it's too convoluted, and multiple API calls are really not that big of an issue for the number of deployments we have of this.

Reply:
No I think it's worth fixing, I'll take a look and push a follow-up commit